ChaosMod/Effects/db/Player/PlayerBigBang.cpp

```diff
@@ -21,10 +21,16 @@ static void OnStart()
 			SET_ENTITY_AS_MISSION_ENTITY(ped, true, true);
 			entities.push_back(ped);
 		}
+		else
+		{
+			maxEntities++;
```

Comment:
Why not just Substract It inside the if? :)

Reply:
Errrr that would work too. Fixed.